python/paddle/fluid/tests/unittests/prim/prim/vjp/static/test_comp_matmul_double_grad.py

```diff
@@ -0,0 +1,328 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+import parameterized as param
+
+import paddle
+from paddle.fluid import core
+
+core._set_prim_backward_enabled(True)
+
+# when dim = 1 reshape op will be deleted by backward algorithm ,
+# it's better to use matmul_grad in static composite pattern
+# batched matrix * batched matrix 4 for trans out.shape = (2, 3, 5)
+# batched matrix * broadcasted vector out.shape = (2, 3)
+# batched matrix * broadcasted matrix out.shape = (2, 3, 5, 4)
+
+TOLERANCE = {
+    "float16": {"rtol": 1e-3, "atol": 1e-3},
+    "float32": {"rtol": 1e-6, "atol": 1e-6},
+    "float64": {"rtol": 1e-15, "atol": 1e-15},
+}
+
+
+@param.parameterized_class(
+    ('primal0', 'primal1', 'primal2', 'trans_0', 'trans_1', 'dtype'),
+    [
+        # (
```

Comment:
Add to do here to uncomment it when we fix static mode

Reply:
done